_assets/ci/Jenkinsfile.unix

```
@@ -0,0 +1,123 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.9.6'
```

nit: might as well use this opportunity to bump to latest tag 
```suggestion
library 'status-jenkins-lib@v1.9.10'
```